SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2066,8 +2066,10 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
       }
     }
 
-    if ((Boundary == HEAT_FLUX) || (Boundary == ISOTHERMAL) || (Boundary == CHT_WALL_INTERFACE)) {
-      /*--- Forces initialization at each Marker ---*/
+    if ((Boundary == HEAT_FLUX) ||
+       (Boundary == ISOTHERMAL) ||
+       (Boundary == CHT_WALL_INTERFACE) || 
+       (Boundary == SMOLUCHOWSKI_MAXWELL)) {       /*--- Forces initialization at each Marker ---*/
```

Comment:
@fmpmorgado please don't do this, modify "CConfig::GetViscous_Wall()" to include the new marker and use it instead.
This repetition of enums throughout the code is a major source of bugs and it is impossible to maintain, see #770.

Reply:
Thank you for the comment. I believe it is working now

Reply:
The compilation problem is solved but you will still need to add SMOLUCHOWSKI_MAXWELL to that function.